index.html

```diff
@@ -154,6 +154,12 @@ <h2>Folders</h2>
 
     color: rgba(255,255,255,0);
 
+}
+
+.new-class {
+    font-size: 18px;
+    width: 100%;
+    max-width: 300px;
```

Comment:
Eu posso aprovar o PR ou solicitar alteração este processo se chama de Review.

Reply:
The implementation is done!